jvm/src/test/scala/uk/gov/homeoffice/drt/db/ABFeatureDaoSpec.scala

```diff
@@ -0,0 +1,78 @@
+package uk.gov.homeoffice.drt.db
+
+import org.specs2.mutable.Specification
+import slick.dbio.DBIO
+
+import scala.concurrent.Await
+import scala.concurrent.duration.DurationInt
+import slick.jdbc.PostgresProfile.api._
+
+import java.sql.Timestamp
+import java.time.Instant
+
+class ABFeatureDaoSpec extends Specification {
```

Comment:
You could use `with Before` so you don't have to manually call `setup` for each test